Framework/Parallel/CMakeLists.txt

```diff
@@ -1,57 +1,28 @@
 set(SRC_FILES
-    src/Communicator.cpp
-    src/ExecutionMode.cpp
     src/IO/Chunker.cpp
     src/IO/EventLoader.cpp
-    src/IO/EventParser.cpp
     src/IO/EventsListsShmemManager.cpp
     src/IO/EventsListsShmemStorage.cpp
     src/IO/MultiProcessEventLoader.cpp
     src/IO/NXEventDataLoader.cpp
     src/IO/EventLoaderChild.cpp
-    src/Request.cpp
-    src/StorageMode.cpp
-    src/ThreadingBackend.cpp
     src/IO/EventLoaderHelpers.cpp
 )
 
 set(INC_FILES
-    inc/MantidParallel/Collectives.h
-    inc/MantidParallel/Communicator.h
-    inc/MantidParallel/ExecutionMode.h
     inc/MantidParallel/IO/Chunker.h
     inc/MantidParallel/IO/EventDataPartitioner.h
     inc/MantidParallel/IO/EventLoader.h
     inc/MantidParallel/IO/EventLoaderHelpers.h
-    inc/MantidParallel/IO/EventParser.h
     inc/MantidParallel/IO/EventsListsShmemManager.h
     inc/MantidParallel/IO/EventsListsShmemStorage.h
     inc/MantidParallel/IO/MultiProcessEventLoader.h
     inc/MantidParallel/IO/NXEventDataLoader.h
     inc/MantidParallel/IO/NXEventDataSource.h
     inc/MantidParallel/IO/PulseTimeGenerator.h
-    inc/MantidParallel/Nonblocking.h
-    inc/MantidParallel/Request.h
-    inc/MantidParallel/Status.h
-    inc/MantidParallel/StorageMode.h
-    inc/MantidParallel/ThreadingBackend.h
 )
 
-set(TEST_FILES
-    ChunkerTest.h
-    CollectivesTest.h
-    CommunicatorTest.h
-    EventDataPartitionerTest.h
-    EventLoaderTest.h
-    EventParserTest.h
```

Comment:
Right, `EventLoaderTest` and `EventLoadedTest` shouldn't be deleted. I was going to delete `EventLoader` and `EventDataPartitoner` but I left them because they are used by MultiProcessEventLoader which loads nexus files parallelly using the boost::interprocess library